com.github.babluboy.bookworm.json

```diff
@@ -0,0 +1,105 @@
+{
+    "id": "com.github.babluboy.bookworm",
+    "runtime": "org.gnome.Platform",
+    "runtime-version" : "3.24",
+    "sdk": "org.gnome.Sdk",
+    "command": "bookworm",
+    "rename-appdata-file": "com.github.babluboy.bookworm.appdata.xml",
+    "finish-args": [
+            "--share=ipc",
+            "--socket=x11",
+            "--device=dri",
+            "--socket=wayland",
+            "--filesystem=home",
+            "--talk-name=ca.desrt.dconf",
+            "--filesystem=xdg-run/dconf",
+            "--filesystem=~/.config/dconf:ro",
+            "--env=DCONF_USER_CONFIG_DIR=.config/dconf"
+    ],
+
+    "build-options": {
+        "cflags": "-O2",
+        "cxxflags": "-O2"
+    },
+    "modules": [
+         {
+                "name": "openjpeg",
+                "buildsystem" : "cmake",
```

Comment:
Not a blocker, but if it works, using cmake-ninja for all these is faster

Reply:
@nedrichards I have heard that ninja is comparatively faster and modern than make  - will check if it supports all the installation requirements of bookworm including the dynamic setting of install path in code, then hopefully will adopt ninja